paasta_tools/utils.py

```diff
@@ -2584,6 +2584,9 @@ def get_uwsgi_exporter_sidecar_image_url(self) -> str:
     def default_should_run_uwsgi_exporter_sidecar(self) -> bool:
         return self.config_dict.get("default_should_run_uwsgi_exporter_sidecar", False)
 
+    def default_should_setup_piscina_prometheus_scraping(self) -> bool:
```

Comment:
reminder to also remove this function once the code calling it has been cleaned up